coral/src/app/components/documentation/DocumentationView.tsx

```diff
@@ -0,0 +1,23 @@
+import ReactMarkdown from "react-markdown";
+import { Box } from "@aivenio/aquarium";
+import classes from "src/app/components/documentation/documentation-view.module.css";
+
+type DocumentationViewProps = {
+  // ⚠️ This component assumes that we're
```

Comment:
We know for sure that we handle sanitized markdown, right? The sanitization is done in the backend? Probably it would be nice to add that info to the comment :o

Reply:
Oh no, we're doing that ourselves, the backend does send stringified html to us. Will update the comment accordingly.

Reply:
Hm but I think I can add a stronger typing for that, too! 
Will check it out.

Reply:
Updated, it's not 100% safe of course (because typescript can only do so much magic :D) but I think we now have a better documentation on type level, too. 

Also since the responsibility of formatting the data moved to domain, `DocumentationView` does not need any special information anymore. It used rawHtml for rendering before, but does not need this now.